noxfile.py

```diff
@@ -0,0 +1,32 @@
+from pathlib import Path
+
+import nox
+
+RUN_DEPS = ["geopandas", "pint", "dataretrieval"]
+
+FORMAT_DEPS = ["ruff >= 0.3.5"]
+TESTS_DEPS = ["pytest", "coverage"]
```

Comment:
**todo**: Ideally, we want to keep those in sync with the deps listed in `pyproject.toml`.